### PR TITLE
[#11947] Format error toasts

### LINKED ISSUE #11947: Improve UI of Error Toasts
- **Environment**:  `master` branch at commit 39192f4

**Description of feature/enhancement**
Propose to update the front-end to facilitate error messages to be represented as multiple toasts rather than a single one.

**Justification**
Currently if there are multiple error messages, it appears in a single toast in certain cases (e.g. where the back-end returns an array of error messages). Since the messages are displayed as an array it reduces the legibility/understanding of errors. Furthermore, the users may also view it as a bug since it seems like the array of messages is returned without being processed. 

Image below shows an instance where there are three error messages:
image

### PR DESCRIPTION
**Pull request by domoberzin**
Fixes #11947

**Outline of Solution**

This PR improves the appearance of error toasts by formatting them to separate each error into a new line, by adjusting the `whitespace` CSS property on the toast component. 

Additionally, due to the adjustment of this property, some of the status messages within the toasts turn up with odd spaces and line breaks. This PR also adds the necessary fixes for those messages.

Lastly, due to the modification, some E2E tests also fail with the odd errors unrelated to toast itself. I found that adding a fixed timeout and closing the toast popup after verification of the message fixes this issue. These changes are also within this PR.

### CODE REVIEW

File: src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts [hunk 1]
```diff
@@ -499,8 +499,8 @@ export class InstructorCourseEditPageComponent implements OnInit {
             newDetailPanels.originalPanel = JSON.parse(JSON.stringify(newDetailPanels.editPanel));
 
             this.instructorDetailPanels.push(newDetailPanels);
-            this.statusMessageService.showSuccessToast(`"The instructor ${resp.name} has been added successfully.
-          An email containing how to 'join' this course will be sent to ${resp.email} in a few minutes."`);
+            this.statusMessageService.showSuccessToast(`"The instructor ${resp.name} has been added successfully. `
+            + `An email containing how to 'join' this course will be sent to ${resp.email} in a few minutes."`);
```
Comment (weiquu) [inline src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts:503]: Think there's an extra inverted commas (" ") for this one!

**Review by weiquu — CHANGES_REQUESTED** [review]
One small nit, otherwise looks good!

**Review by jasonqiu212 — APPROVED** [review]
LGTM!

**Review by weiquu — APPROVED** [review]
LGTM! Thanks for contributing and good job for the research into the E2E issues (:

**Pull request merged**